algo/src/test/java/com/hartwig/serve/sources/ckb/ActionableEvidenceFactoryTest.java

```diff
@@ -293,4 +314,38 @@ private static CkbEvidenceBlacklistModel createBlacklistModel(@NotNull CkbBlackl
 
         return CkbBlacklistTestFactory.createSpecificEvidenceBlacklist(entry);
     }
+
+    @NotNull
+    private static Reference createReference(String year) {
+        return ImmutableReference.builder()
+                .id(0)
+                .pubMedId("123456")
+                .title("Title")
+                .shortJournalTitle("Short Journal Title")
+                .pages("1-2")
+                .abstractText("Abstract Text")
+                .url("http://www.url.com")
+                .journal("Journal")
+                .authors("Authors")
+                .volume("Volume")
+                .issue("Issue")
+                .date("Date")
+                .year(year)
+                .build();
+    }
+
+    @NotNull
+    private static Therapy createTherapy(int createYear) {
+        return ImmutableTherapy.builder()
+                .id(0)
+                .createDate(LocalDate.of(createYear, 1, 1))
+                .updateDate(LocalDate.of(2021, 1, 1))
+                .therapyName("Therapy")
+                .drugs(List.of())
+                .synonyms(List.of())
+                .globalApprovalStatuses(List.of())
+                .description("Description")
+                .references(List.of())
+                .build();
+    }
```

Comment:
Did you consider moving these functions into `CkbTestFactory`? 

This factory currently mainly creates entries, so may not be a good fit but at least something to consider